utils/Cargo.toml

```diff
@@ -18,6 +18,7 @@ lz4-sys = "1.9.2"
 serde = { version = ">=1.0.27", features = ["serde_derive", "rc"] }
 serde_json = ">=1.0.9"
 sha2 = "0.9.1"
+zstd-sys = "2.0.1"
```

Comment:
Thanks for the first contribution! I wonder why not use `zstd` crate directly?

Reply:
The main reason is that the lz4 using the `lz4-sys` crate, so I use the `zstd-sys` here, and feels like it would be more flexible. Should I change to `zstd`? Thanks for your review!

Reply:
`zstd` -> `zstd-safe` -> `zstd-sys`

Seems no flexible requirement here, using a safe crate directly should be better. :)

Reply:
Ok.